namespaces/live.cloud-platform.service.justice.gov.uk/laa-crown-court-remuneration-production/resources/rds.tf

```diff
@@ -0,0 +1,111 @@
+module "rds-instance" {
+  source   = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=migration"
```

Comment:
for the version, please use 

source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=7.0.0"